Mathlib/Data/Finsupp/Weight.lean

```diff
@@ -105,6 +105,13 @@ theorem NonTorsionWeight.ne_zero [NonTorsionWeight w] (s : σ) :
   apply Nat.zero_ne_one.symm
   exact NonTorsionWeight.eq_zero_of_smul_eq_zero h
 
+variable {w} in
+lemma weight_sub_single_add {f : σ →₀ ℕ} {i : σ} (hi : f i ≠ 0) :
```

Comment:
I think we can keep `w` explicit.

Reply:
That would mean `weight_sub_single_add hi` below will become `weight_sub_single_add _ hi` or `weight_sub_single_add w hi` ...

Reply:
OK, no problem.

Reply:
I assumed that you mean keeping {w} implicit is no problem, but now I realize you might mean adding an underscore is no problem ... Can you clarify and should I make a PR to change anything?